app/assets/javascripts/admin/addon/components/ip-lookup.gjs

```diff
@@ -0,0 +1,266 @@
+                    @icon="xmark"
+                    class="btn-transparent"
+                  />
+                {{/if}}
+
+              </div>
+            </div>
+            <dl>
+              {{#if this.location}}
+                {{#if this.location.hostname}}
+                  <dt>{{i18n "ip_lookup.hostname"}}</dt>
+                  <dd>{{this.location.hostname}}</dd>
+                {{/if}}
+
+                <dt>{{i18n "ip_lookup.location"}}</dt>
+                <dd>
+                  {{#if this.location.location}}
+                    <a
+                      href="https://maps.google.com/maps?q={{this.location.latitude}},{{this.location.longitude}}"
+                      rel="noopener noreferrer"
+                      target="_blank"
+                    >
+                      {{this.location.location}}
+                    </a>
+                  {{else}}
+                    {{i18n "ip_lookup.location_not_found"}}
+                  {{/if}}
+                </dd>
+
+                {{#if this.location.organization}}
+                  <dt>{{i18n "ip_lookup.organisation"}}</dt>
+                  <dd>{{this.location.organization}}</dd>
+                {{/if}}
+              {{else}}
+                {{loadingSpinner size="small"}}
+              {{/if}}
+
+              <dt class="other-accounts">
+                {{i18n "ip_lookup.other_accounts"}}
+                <span class="count">{{this.totalOthersWithSameIP}}</span>
+                {{#if this.otherAccounts}}
+                  <DButton
+                    @action={{this.deleteOtherAccounts}}
+                    @icon="triangle-exclamation"
+                    @translatedLabel={{i18n
+                      "ip_lookup.delete_other_accounts"
+                      count=this.otherAccountsToDelete
+                    }}
+                    class="btn-danger pull-right"
+                  />
+                {{/if}}
+              </dt>
+
+              <ConditionalLoadingSpinner
+                @size="small"
+                @condition={{this.otherAccountsLoading}}
+              >
+                {{#if this.otherAccounts}}
+                  <dd class="other-accounts">
+                    <table class="table table-condensed table-hover">
+                      <thead>
+                        <tr>
+                          <th>{{i18n "ip_lookup.username"}}</th>
+                          <th>{{i18n "ip_lookup.trust_level"}}</th>
+                          <th>{{i18n "ip_lookup.read_time"}}</th>
+                          <th>{{i18n "ip_lookup.topics_entered"}}</th>
+                          <th>{{i18n "ip_lookup.post_count"}}</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        {{#each this.otherAccounts as |a|}}
+                          <tr>
+                            <td class="user">
+                              <LinkTo @route="adminUser" @model={{a}}>
```

Comment:
`a` is as concise as it gets, but I got confused when reading quickly below, I thought `a.post_count` was an anchor. 🤣